internal/k8s/api/api.go

```diff
@@ -0,0 +1,23 @@
+package api
+
+type Apis struct {
```

Comment:
could you add the comments to the new exported types and their exported methods? this will follow the code style

Reply:
I'll put a comment or two where I thinks it's needed, if that's okay. I removed quite a few comments like:
```
\\ GetPods gets the pods
fun GetPods() {
```

Reply:
Some methods are self documenting, i.e.
```
func NewApis() *Apis {
	return &Apis{}
}
```

Reply:
updated.

Reply:
> As the comment for this would just be the signature of the method in English.

some will probably do
the existing code across the packages follows that convention - exported types with their exported methods are documented. the new code - doesn't, so it's inconsistent

Reply:
If consistency is an issue, I'm more than happy to remove the redundant comments in the rest of the codebase.

Reply:
I agree that they should have a comment, it's a common best practice https://github.com/golang/go/wiki/CodeReviewComments#doc-comments